var/spack/repos/builtin/packages/occa/package.py

```diff
@@ -50,13 +51,16 @@ class Occa(Package):
             description='Activates support for OpenCL')
     variant('coi',   default=False,
             description='Activates support for COI')
-    depends_on('cuda', when='+cuda')
-    
+    # depends_on('cuda', when='+cuda')
```

Comment:
We can do a better job of instructing the user what to do for packages like this.  What they probably want to do is add an [external CUDA](https://spack.readthedocs.io/en/latest/build_settings.html#external-packages).  If they do this, the Spack CUDA package is just a stub, so you could try adding an external on your mac and on Linux.  e.g., in `packages.yaml`:

```yaml
packages:
    cuda:
        paths:
            cuda@7.5.18: /usr/local
```